packages/common/src/services/local-storage/LocalStorage.ts

```diff
@@ -47,6 +48,23 @@ export class LocalStorage {
     return null
   }
 
+  async getExpiringJSONValue<T>(key: string): Promise<T | null> {
+    const res: Nullable<{ value: T; expiry: number }> = await this.getJSONValue(
```

Comment:
nit: I personally prefer the type parameter in the method call vs the variable type:
`const res = await this.getJSONValue<{value: T; expiry: number} | null>(key)`

Reply:
fair! I'm more of a fan of the former, it usually feels odd to me to pass a type itself as a param